synthtool/gcp/common.py

```diff
@@ -326,6 +326,36 @@ def node_library(self, **kwargs) -> Path:
 
         return self._generic_library("node_library", **kwargs)
 
+    def node_mono_repo_library(self, relative_dir, **kwargs) -> Path:
```

Comment:
since this logic is pretty much identical to `def node_library`, I might just make `node_library` take an optional `relative_dir` parameter, and then default to `./` unless there's an alternate `relative_dir` provided.

Reply:
Normally, I think it would be better to simply modify the existing code rather than duplicating it. However, I think in this case it's better to duplicate it, because: 1) it makes it easier to read and debug (just follow the node_mono_repo naming), and 2) we are eventually migrating to a mono repo, and eventually all the split repo paths will be deprecated. Given that it's only a few helper methods that are truly being copied over, I'd opt for copying over adding a new path to a helper method (this goes for other helper methods that are similar as well). Furthermore, if there are more specific changes we need to make for mono-repos, it will be easier to add to a mono-repo only path rather than making sure it doesn't affect split repos.